docker-compose.yml

```diff
@@ -2,20 +2,23 @@ version: "3"
 services:
     sonar:
         image: sonarqube:community
+        container_name: sonar
         ports:
             - "9000:9000"
         volumes:
             - shared-volume:/mnt/data
 
     visualization:
         image: codecharta/codecharta-visualization
+        container_name: visualization
```

Comment:
I'd like to keep the `codecharta` prefix. Otherwise, `visualization` could be ambigious if there are other containers with the same name on the users environment.